app/invocation/invocation_execution_card.tsx

```diff
@@ -218,27 +216,27 @@ export default class ExecutionCardComponent extends React.Component {
     }
   }
 
-  handleInputChange(event: any) {
-    const target = event.target;
+  handleInputChange(event: React.ChangeEvent) {
```

Comment:
I think you could eliminate the casting here and below with something like
```
React.ChangeEvent<HTMLInputElement>
```

Reply:
Good catch, I was confusing this with `MouseEvent` where `target` doesn't wind up having the expected type even when a type param is passed. Fixed